src/js/burials/config/form.js

```diff
@@ -523,12 +530,18 @@ const formConfig = {
             }), {
               'ui:required': form => form.burialAllowanceRequested === 'service',
             }),
-            transportationReceipts: _.assign(fileUploadUI('Documentation for transportation of the Veteran’s remains', {
+            transportationReceipts: _.assign(fileUploadUI('Documentation for transportation of the Veteran’s remains or other supporting evidence', {
               addAnotherLabel: 'Add Another Document',
               hideIf: form => _.get('view:claimedBenefits.transportation', form) !== true
             }), {
               'ui:required': form => _.get('view:claimedBenefits.transportation', form) === true,
```

Comment:
Thanks @stacyhlinn, the logic for conditionally requiring document upload was already in place.